website/en/02-assembly.md

```diff
@@ -9,14 +9,14 @@ Just like web browsers hide the differences between Windows/macOS/Linux, operati
 In this book, I chose RISC-V as the target CPU because:
 
 - [The specification](https://riscv.org/technical/specifications/) is simple and suitable for beginners.
-- It's a trending ISA (Instruction Set Architecture) in recent years, along with x86 and Arm.
+- It's a trending ISA (Instruction Set Architecture) in recent years, along with x86 and ARM.
```

Comment:
`Arm` is intentional to follow how it is called today by [arm](https://www.arm.com/architecture/cpu/a-profile):

```suggestion
- It's a trending ISA (Instruction Set Architecture) in recent years, along with x86 and Arm.
```

Reply:
Ugh, that seems like a bad choice on their part, but I updated to match this.

Reply:
IMO "ARM" was cool btw 🤫